src/components/Alert/Alert.vue

```diff
@@ -1,19 +1,41 @@
 <template>
   <div
-    :class="['w-full p-4.5 rounded-md flex justify-center align-center',
-             { 'bg-turquoise-100': info },
-             { 'bg-mint-100': success },
-             { 'bg-lemon-100': warning },
-             { 'bg-coral-100': error }
+    :class="['w-full border-l-4 p-4.5 rounded-r-lg flex justify-start align-center font-light',
+             { 'bg-turquoise-100 border-turquoise-500': info },
+             { 'bg-mint-100 border-success': success },
+             { 'bg-lemon-100 border-warning': warning },
+             { 'bg-coral-100 border-coral-700': error }
     ]"
   >
+    <Info
```

Comment:
here we could use the `<component :is="computedIconName"` trick

Reply:
+1